apps/mochi-web/components/settings/general/MoneySource.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Button,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+  Typography,
+} from '@mochi-ui/core'
+import { PlusLine } from '@mochi-ui/icons'
+import { Control, Controller } from 'react-hook-form'
+import { GeneralFormValue } from './types'
+
+interface Props {
+  control: Control<GeneralFormValue>
+}
+
+export const MoneySource = ({ control }: Props) => {
+  return (
+    <Controller
+      name="defaultMoneySource"
+      control={control}
+      render={({ field, fieldState }) => (
+        <FormControl error={!!fieldState.error} className="min-w-[160px]">
+          <FormLabel>Default money source</FormLabel>
+          <Select {...field}>
+            <SelectTrigger className="justify-between border border-divider">
+              <SelectValue placeholder="Mochi wallet" />
+            </SelectTrigger>
+            <SelectContent>
+              <SelectItem value="mochi">Mochi wallet</SelectItem>
+              <SelectItem value="evm">EVM | 0xfsf...few</SelectItem>
+              <SelectItem value="sol">SOL | 0xfsf...few</SelectItem>
+              <Button
+                variant="ghost"
+                color="white"
+                className="w-full pl-2 pr-2"
+              >
+                <PlusLine className="w-4 h-4" />
+                <Typography
```

Comment:
Why do we need to use Typography here?

Reply:
Can we resemble the UI?

<img width="291" alt="image" src="https://github.com/consolelabs/mochi-ui/assets/12707960/da0b4a63-c55e-4766-b20f-bd26996e92fb">